src/models/User.ts

```diff
@@ -135,7 +135,7 @@ export class User {
     const email = this.email || '';
 
     if (!validateEmail(email) || !this.validatePassword()) {
-      this.errors.push('Login failed');
+      return null;
```

Comment:
Returned null, because the signinController that is calling this method is not checking for user.errors, but just wants to know if the response is null or not.